src/DebugBar/Resources/debugbar.js

```diff
@@ -425,7 +425,9 @@ if (typeof(PhpDebugBar) == 'undefined') {
             this.activeDatasetId = null;
             this.datesetTitleFormater = new DatasetTitleFormater(this);
             this.options.bodyMarginBottomHeight = parseInt($('body').css('margin-bottom'));
-            this.isIframe = window.self !== window.top;
+            try {
+                this.isIframe = window.self !== window.top && window.top.phpdebugbar;
+            } catch (error) {}
```

Comment:
Should the catch set the variable to false?

Reply:
I did add it
but by default wouldn't it be `undefined`? 
And a conditional on `undefined` results in `false`, right?